include/highfive/bits/H5Node_traits.hpp

```diff
@@ -112,6 +112,21 @@ class NodeTraits {
     /// \return the name of the object
     std::string getObjectName(size_t index) const;
 
+    ///
+    /// \brief return the path to the current object
+    /// \return the path to the object
+    std::string getPath() const;
+
+    ///
+    /// \brief moves an object and its content within an HDF5 file.
+    /// \param src_path relative path of the object to current File/Group
+    /// \param dest_path new relative path of the object to current File/Group
+    /// \param parents if true necessary intermediate groups are created. Default: true
+    /// \return boolean that is true if the move was successful
+    bool moveObject(const std::string& src_path,
```

Comment:
Personal nitpick: I find the `Object` a little redundant, could this be just `move`, or better `rename`, since I don't think that any of the data is actually moved?

Reply:
`rename` sounds interesting but for me if I saw for a first time the `File/Group` method `rename` (or `move`) then I would think that I'm renaming (moving) this `File/Group`. How do you think?

Reply:
`move` might in the end be closed to the underlying `H5Lmove`. On the other-hand `rename` might be closest to the description of `H5Lmove`: "Renames a link within an HDF5 file.". `moveObject` is a bit a name on its own. Probably I would vote for `rename`. Or if you're hesitant that it is not sufficiently clear, I could still suggest `renameLink` which is short of the description of the underlying function.

Reply:
I can do as you want. It doesn't matter a lot for me
Everyone wants it is to be called `rename`?

Reply:
Yes, please.